operations/security_and_authentication/Add_LDAP_User_Federation.md

```diff
@@ -271,13 +275,13 @@ LDAP user federation is not currently configured in Keycloak. For example, if it
       > **NOTE:** A properly configured Docker or Podman environment is required.
 
       ```bash
-      ncn-m001# /mnt/pitdata/${CSM_RELEASE}/hack/load-container-image.sh dtr.dev.cray.com/zeromq/zeromq:v4.0.5
+      ncn-m001# ${CSM_DISTDIR}/hack/load-container-image.sh dtr.dev.cray.com/zeromq/zeromq:v4.0.5
       ```
 
    3. Re-encrypt the existing secrets:
 
       ```bash
-      ncn-m001# /mnt/pitdata/prep/site-init/utils/secrets-reencrypt.sh customizations.yaml \
+      ncn-m001# ./utils/secrets-reencrypt.sh customizations.yaml \
       /mnt/pitdata/prep/site-init/certs/sealed_secrets.key /mnt/pitdata/prep/site-init/certs/sealed_secrets.crt
```

Comment:
Still referencing /mnt/pitdata/prep/site-init here.  The full command should be
`./utils/secrets-reencrypt.sh customizations.yaml ./certs/sealed_secrets.key ./certs/sealed_secrets.crt`